src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
@caseypt makes a great point.  It looks like for "read only" in share mode we are checking for a user match (https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/models.js#L153).  Could that call be extracted into a function `isReadOnly` that checks either that the user matches or that a new `readOnly` attribute has been set to serve both purposes?  Would be a much lighter solution that the clone, and give us the ability to decide later if we should, in fact, be persisting any of these values.